### DISCUSSION TITLE
Enable quick compaction-checker

### DISCUSSION THREAD

**Discussion by xiaobiaozhao** [apache/kvrocks#1283] [General]
In some case，we need save a lot of data with short ttl, eg ttl = 3600s.
In current kvrocks, it takes at least two days for compation to take place, which can cause disk space not to be released quickly. I think kvrocks needs to come up with a configuration to figure out how long compation will take.

New a conf item `compaction-checker-force-compact-seconds` https://github.com/apache/incubator-kvrocks/blob/c439f3af9d5c7ccb97ff4c7bdab088287acd73cc/src/storage/compaction_checker.cc#L58

And new a conf item `compaction-checker-interval-seconds`
https://github.com/apache/incubator-kvrocks/blob/c439f3af9d5c7ccb97ff4c7bdab088287acd73cc/src/server/server.cc#L184

**Comment by torwig** [comment]
Since 2 days is a significantly long time, introducing that interval as a setting in the configuration file makes sense to me.
